### DISCUSSION TITLE
Windows key action on keyboard

### DISCUSSION THREAD

**Discussion by saucyknave** [Open-Shell/Open-Shell-Menu#1558] [General]
A recent Windows 11 update must have changed something, because now pressing the Windows key on my keyboard is opening the Windows 11 Start Menu instead of the OpenShell start menu.  I'm just posting this in case someone has some insight or if more people are seeing the same thing.  The settings are still good, but they're being ignored for that key.

**Comment by yahyaten** [comment]
I have the same problem, I installed the latest beta version and then reinstalled the latest stable version, and the issue was resolved for me.

**Reply by saucyknave** [reply]
Yahyaten, thanks for the reply.  It finally occurred to me to check Issues to see if this is a known thing and of course it is.  :)  I'm just about to go down the same Stable->Beta->Stable path, too.

**Comment by atchub21** [comment]
Win11 Ver 22621.1848, the Start Keyboard button does not open open shell at all, opens the builtin windows 11 menu.
Tried 4.4.181 and 4.4.170

**Reply by ge0rdi** [reply]
Please, make sure you are using [latest](https://github.com/Open-Shell/Open-Shell-Menu/releases) Open-Shell build on Windows 11.
Official release doesn't support Windows 11 yet.

**Reply by atchub21** [reply]
thanks.   4.4.189 release fixed it.

**Comment by richardlucas761** [comment]
I had the same issue but 4.4.189 fixed it.

As a side note, selecting the "Open Shell Update" updated me to an earlier version which still had the issue, perhaps this is my problem and I should have selected the "nightly builds" option from the update program.

**Reply by ge0rdi** [reply]
> I should have selected the "nightly builds" option from the update program

That's right.
`Nightly builds` setting checks for any new build (including pre-release ones).

Without the setting updater is checking only official releases.
